kitchen.yml

```
@@ -81,16 +86,8 @@ suites:
     provisioner:
       product_name: chef
       product_version: 13.6.4
-  - name: config-chef-13-auth
-    run_list: test::config_auth
-    excludes:
-      - windows-2016
-      - windows-2019
-    provisioner:
-      product_name: chef
-      product_version: 13.6.4
 
-# Testing windows stuff here:
+# # Testing windows stuff here:
```

Single # and keep in line with yaml :+1: 
```suggestion
  # Testing windows stuff here:
```

---

Corrected